kyuubi-rest-client/src/main/java/org/apache/kyuubi/client/BatchRestApi.java

```diff
@@ -46,10 +47,23 @@ public Batch createBatch(BatchRequest request) {
   }
 
   public Batch createBatch(BatchRequest request, File resourceFile) {
+    return createBatch(request, resourceFile, Collections.emptyList());
+  }
+
+  public Batch createBatch(BatchRequest request, File resourceFile, List<String> extraResources) {
     setClientVersion(request);
     Map<String, MultiPart> multiPartMap = new HashMap<>();
     multiPartMap.put("batchRequest", new MultiPart(MultiPart.MultiPartType.JSON, request));
     multiPartMap.put("resourceFile", new MultiPart(MultiPart.MultiPartType.FILE, resourceFile));
+    extraResources.stream()
+        .distinct()
+        .filter(StringUtils::isNotBlank)
+        .map(path -> Paths.get(path).toFile())
+        .filter(File::exists)
```

Comment:
do not need to filter, throw RuntimeException if not exists.

Reply:
OK. Fixed.